src/popup/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -1,67 +1,62 @@
 <template>
   <div class="token-details">
-    <Plate class="token-header">
-      <div class="token-profile">
-        <Loader v-if="loading" />
-        <Tokens
-          :tokens="
-            tokenPairs.token0 && tokenPairs.token1
-              ? [tokenPairs.token0, tokenPairs.token1]
-              : [tokenData]
-          "
-          :symbol-length="22"
-        />
-        <TokenAmount
-          :amount="convertedBalance"
-          no-symbol
-          :aex9="id !== AETERNITY_CONTRACT_ID"
-        />
-      </div>
-      <div class="token-actions">
-        <BtnBox
-          class="token-actions-btn"
-          :disabled="!convertedBalance"
-          @click="openTransferSendModal()"
-        >
-          <SendIcon />
-          {{ $t('pages.token-details.send') }}
-        </BtnBox>
-        <BtnBox
-          class="token-actions-btn"
-          @click="openTransferReceiveModal()"
-        >
-          <ReceiveIcon />
-          {{ $t('pages.token-details.receive') }}
-        </BtnBox>
-        <BtnBox
-          v-if="id === AETERNITY_CONTRACT_ID"
-          class="token-actions-btn"
-          :href="SIMPLEX_URL"
-        >
-          <BuyIcon />
-          {{ $t('pages.fungible-tokens.buyAe') }}
-        </BtnBox>
-      </div>
-      <div
-        slot="bottom"
-        class="token-tabs"
+    <Loader v-if="loading" />
+
+    <Tokens
+      :tokens="
+        tokenPairs.token0 && tokenPairs.token1
+          ? [tokenPairs.token0, tokenPairs.token1]
+          : [tokenData]
+      "
```

Comment:
Maybe we should extract this logic to computed properties?

Reply:
Maybe, but I don't want to touch the logic as this was not in the scope of the task.